_posts/research/2020-04-05-robotic-skin.md

```diff
@@ -0,0 +1,92 @@
+
+1. **Kinematic Calibration:** we present a framework for automatic kinematic calibration that leverages an IMU to automatically locate and orient skin units, of arbitrary number, along the surface of a robot.
+
+1. **Control Example:** we demonstrate an example control scenario where a robot arm utilizes proximity sensors embedded within a skin unit to **avoid collision** when an object is nearby. We impose a repulsive force on the point the skin sensor is mounted on the robotic arm to ensure it will not collide with an object, but will still continue on a specified trajectory.
+
+
+## Skin Units
+
+Skin units are embedded with an IMU for kinematic calibration and a proximity sensor to sense nearby objects. We chose to embed our circuit in a flexible polymer called ecoflex in order to have each skin unit conform to the shape of the surface it is placed on. The figure below shows the flexible capabilities of the skin sensor and an image of one unit mounted on a robotic arm.
+
+
+<div class="row">
+  <div class="col-md-6 col-print-6">
+    {% include image.html url="research/roboskin/skin_unit.jpg" max-width="75%" description="Flexible skin unit mounted on a robotic arm." %}
+  </div>
+  <div class="col-md-6 col-print-6">
+    {% include image.html url="research/roboskin/bent_skin_unit.jpg" max-width="75%" description="Demonstration of the skin’s flexible capabilities." %}
+  </div>
+</div>
+
+
+## Kinematic Calibration
+
+To automatically locate skin units along the surface of a robot, we utilize an IMU that provides measurements of angular velocity and linear acceleration. Our optimization algorithm estimates the position and orientation of skin units using Denavit-Hartenberg as demonstrated in the figure below.
+
+{% include image.html url="research/roboskin/calibration.png" max-width="75%" description="Illustrated setup of skin units (S) placed along the robots links (L) that are separated by joints (J). We estimate the  Denavit-Hartenberg parameters of each joint in order to calculate the pose of each skin unit along the surface of the robot. " %}
+
+
+
+## Control Example
+
+Calibrated skin units can be used to locate obstacles with respect to the robot’s links and avoid undesired collisions by exerting a repulsive force on the robotic arm. To avoid object we require the following information:
+
+* Calibrated pose of the Skin Unit (x, y, z, φ, θ, ψ)
+* Distance d normal to the skin sensor
+
+With this information, we apply a repulsive force perpendicular to the skin unit as seen in the figure below.
+
+{% include image.html url="research/roboskin/control.png" max-width="75%" description=" A mounted skin unit exerting a repulsive force β on a robot's trajectory
+**T** when an object is near the skin unit. The distance d is measured in millimeters. As an object approaches, β increases exponentially." %}
```

Comment:
Why is $\beta$ applied to 2 directions ? 
In addition, the coordinate systems should be separated. The $world$ frame should be better attached to the robot's base and the IMU coordinate should be placed close to the IMU. So that way we know which frame refers to which object. It would be also good to have a description in here.

Reply:
agreed